src/main/js/helpers/legend.js

```diff
@@ -70,15 +70,17 @@ const loadLegendItem = (legendSVG, t, shownTargets, eventHandlers) => {
     validateLegendLabel(t.label);
     const text = getText(t.label.display);
     const index = shownTargets.indexOf(t.key);
+    const shouldForceDisableLegendItem =
+        !!t.label.isDisabled || utils.isEmptyArray(t.values);
     const itemPath = legendSVG
         .append("li")
         .classed(styles.legendItem, true)
-        .attr("aria-selected", index > -1)
-        .attr("aria-disabled", !!t.label.isDisabled)
+        .attr("aria-selected", shouldForceDisableLegendItem || index > -1)
```

Comment:
So when there is no data, this will be force-set to `true`? Is that what you want to do here? 
What happens in a scenario where, you have:
* Data is present from Range 0-40
* No Data is present from Range 41-50
* Data is present from Range 51-100

and you deselect your data set at 0-40? Just curious

Reply:
This code is confirming the style of the disabled element is the same across the application. 

Previously, When there is no data, by default (only in paired-result) 'aria-selected' going to be false. So, even after setting the disabled style, the icon shade used to be like unselected, without the shade in it. like below:
<img width="113" alt="Screen Shot 2019-08-12 at 6 54 40 PM" src="https://user-images.githubusercontent.com/53594676/62868252-d28d4980-bd32-11e9-9ebc-edb7518de7d5.png">
Where as the disabled icons in the other screens will be like below:
<img width="109" alt="Screen Shot 2019-08-12 at 6 56 46 PM" src="https://user-images.githubusercontent.com/53594676/62868321-fd779d80-bd32-11e9-8b74-a6b8017c99a9.png">

To avoid this we have added this code change giving selected style along with the disabled.

When there is data, shouldForceDisableLegendItem will be false. So, it will not impact if you select/deselect.

Hope I have answered your question. Let me know if any.

Thanks,

Reply:
Sounds good! Did I miss the unit test for this? I can see one for `aria-disabled`.